### PR TITLE
Update the Starter Guide for React

### PR DESCRIPTION
**Pull request by tobiasthorin**
Issue: N/A

## What I did
- Correct some grammar.
- Change some language to be more clear and accommodating.
- Remove the inclusion of the actions add on in the code example, since this prevents successful compilation if the guide if followed exactly.

## How to test
- Follow the steps in the React Starter Guide
- Storybook should now start without problems on the first try

### CODE REVIEW

File: docs/src/pages/basics/guide-react/index.md [hunk 1]
```diff
@@ -5,41 +5,41 @@ title: 'Storybook for React'
...
 
 ## Add react, react-dom, babel-core, and babel-loader
 
-Make sure that you have `react`, `react-dom`, `@babel/core`, and `babel-loader` in your dependencies as well because we list these as a peerDependency:
+Make sure that you have `react`, `react-dom`, `babel-core`, and `babel-loader` in your dependencies as well because we list these as a peer dependencies:
 
 ```sh
 npm i --save react react-dom
-npm i --save-dev @babel/core
+npm i --save-dev babel-core
```
Comment (ndelangen) [inline docs/src/pages/basics/guide-react/index.md]: Is this right?

For babel 7 this should be ` @babel/core` right?

**Review by ndelangen — CHANGES_REQUESTED** [review]

**Comment by ndelangen** [comment]
Thanks @tobiasljungstrom can you help me understand the babel dependency change?

File: docs/src/pages/basics/guide-react/index.md (hunk 1 above)
Comment (tobiasljungstrom) [reply docs/src/pages/basics/guide-react/index.md]: That's right. A mistake on my part, well caught.

**Review by ndelangen — APPROVED** [review]

File: docs/src/pages/basics/guide-react/index.md [hunk 2]
```diff
@@ -5,41 +5,41 @@ title: 'Storybook for React'
...
+First of all, you need to add `@storybook/react` to your project. To do that, run:
 
 ```sh
 npm i --save-dev @storybook/react
 ```
 
 ## Add react, react-dom, babel-core, and babel-loader
 
-Make sure that you have `react`, `react-dom`, `@babel/core`, and `babel-loader` in your dependencies as well because we list these as a peerDependency:
+Make sure that you have `react`, `react-dom`, `babel-core`, and `babel-loader` in your dependencies as well because we list these as a peer dependencies:
```
Comment (igor-dv) [inline docs/src/pages/basics/guide-react/index.md]: Why babel-core ?

Comment (tobiasljungstrom) [reply docs/src/pages/basics/guide-react/index.md]: Damn, that was a mistake that carried over from an old version. It was fixed on line 38, but we missed it here.

Comment (tobiasljungstrom) [reply docs/src/pages/basics/guide-react/index.md]: And on line 19. :|

Comment (tobiasljungstrom) [reply docs/src/pages/basics/guide-react/index.md]: Yeah, I can make another PR.

**Pull request merged**